### ISSUE TITLE
Add support for multi Repo

### ISSUE THREAD

**Issue by DJDDDM** [abapGit/abapGit#7052]
Please add support for multi repos via allowing the base package of a repository to have Information about the superpackage also serialized. 

Use Case:
For big projects you have independend parts. So each independend part gets its own git repository.
In the source system the packages of these individual repositories are neatly organized in subpackages.
In order for the target sytem to have this nested package structure it is currently necessary to create packages with the corresponding superpackages per hand.
This should be automated.

**Comment by mbtools** [comment]
Yes, it's a nice-to-have feature. There's some effort required to implement this so low priority.

You could create a repo for the main package structure manually (clone https://github.com/abapGit-tests/DEVC_main_sub/tree/main/src). Then deploy this repo first to get all packages into the target system, and remove the repo afterwards (without uninstalling objects, obviously). Now you can bring in all the parts.

**Comment by bkrencker** [comment]
hi there, also interested in this feature :)

**Comment by mbtools** [comment]
I'm happy to implement enhancements like this but I don't have the time to do these for free ([sponsor](https://github.com/sponsors/mbtools)).
